README.md

```diff
@@ -76,7 +76,7 @@ PartyFoul.configure do |config|
   # The OAuth token for the account that is opening the issues on GitHub
   config.oauth_token            = 'abcdefgh1234567890'
 
-  # The API endpoint for GitHub. Unless you are hosting a private
+  # The API api_endpoint for GitHub. Unless you are hosting a private
```

Comment:
Ah, I see you were making it consistent. I apologize, I missed that. Yes, thank you for fixing in the template.